Makefile

```diff
@@ -1,6 +1,16 @@
 # Config
 WORKERS = 4
 SHELL = /bin/bash
+MANAGE := python ./manage.py
```

Comment:
@brousch There isn't really a need to call python explicitly here - it duplicates the shebang from `manage.py`. I would only keep `HONCHO_MANAGE := honcho run ./manage.py`